_includes/scenarios/integrate/establish-a-relationship-to-another-identity.md

```diff
@@ -0,0 +1,168 @@
+
+{% include copy-notice description="Read the `id` of the Relationship from the `relationships.id` property and the `id` of the RelationshipChange from the `relationships.changes.id` property of the synchronization `result` for the next step." %}
+
+### Accept the RelationshipRequest
+
+If the Templator accepts the RelationshipRequest, the `status` of the data object of type [Relationship]({% link _docs_integrate/data-model-overview.md %}#relationship) will change from `"Pending"` to `"Active"` and therefore an active Relationship between the Templator and the Requestor will be established. In addition, the `transport.relationshipChanged` [Connector event]({% link _docs_integrate/connector-events.md %}) will be triggered. To accept the RelationshipRequest, consult the [Accept RelationshipChange]({% link _docs_use-cases/use-case-transport-accept-relationship-change.md %}) use case description and specify the `id` of the Relationship and the `id` of the associated RelationshipChange.
+
+For rejecting the RelationshipRequest and therefore not establishing an active Relationship between the Templator and the Requestor, take a look at the documentation of the [Reject RelationshipChange]({% link _docs_use-cases/use-case-transport-reject-relationship-change.md %}) use case.
+{: .notice--info}
+
+### Be informed about the acceptance of the RelationshipRequest
+
+Assuming the Requestor is a Connector, it must [synchronize the updates of the Backbone]({% link _docs_use-cases/use-case-transport-synchronize-updates-of-backbone.md %}) after the Templator has accepted the RelationshipRequest. The synchronization causes the `transport.relationshipChanged` [Connector event]({% link _docs_integrate/connector-events.md %}) to be triggered and the `result` of the response after synchronization shows in particular that the `status` of the [Relationship]({% link _docs_integrate/data-model-overview.md %}#relationship) has been changed from `"Pending"` to `"Active"` and that the `status` of the associated [RelationshipChange]({% link _docs_integrate/data-model-overview.md %}#relationshipchange) with `"Creation"` as `type` has been changed from `"Pending"` to `"Accepted"`. Now the Requestor is informed that the Templator has accepted the RelationshipRequest and therefore an active Relationship has been established between them. If the Requestor is an App user instead, they are informed about the acceptance of the RelationshipRequest analogously, without having to synchronize manually.
```

Comment:
I'd omit the "without having to synchronize manually", since in a productive environment we encourage the usage of the Sync Module such that the Connector wouldn't synchronize manually, either.